### PR TITLE
bash_completion

### PR DESCRIPTION
**Pull request by Fuseteam**
so uh this completes for the options install, update, remove and list that is only the first argument to ubports-qa not sure if there is a reasonable way to complete for repo and pr number yet
also not sure how to "install" it but it should go into /etc/bash_completion.d/

### CODE REVIEW

**Comment by Fuseteam** [comment]
because it makes use of ubports-qa list to autocomplete the argument for remove it cuts off the branch name incorrectly when it's of the form xenial_-_PR_repo_number i.e. repo and number are completely cut off

**Comment by NotKit** [comment]
Seems to work fine for me, except for the mentioned xenial_-_PR_ case, which is ubports-qa list issue

File: ubports-qa_completion [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    local cur prev
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    case $COMP_CWORD in
+	    1) COMPREPLY=( $(compgen -W "$(ubports-qa -h |awk '/   install/,/    update/{print $1}')" -- "${cur}") );;
+	    2)  [ "${prev}" = "remove" ] && COMPREPLY=( $(compgen -W "$(ubports-qa list)" -- "${cur}") );;
+	    *);;
+    esac
+    case $cur in
+	    -*) COMPREPLY=( $(compgen -W "$(_parse_usage ubports-qa)" -- "${cur}") );;
```
[the remark this answers is not in the document]
Comment (Fuseteam) [reply ubports-qa_completion:13]: no i don't think this handle --arguments atm

**Comment by Fuseteam** [comment]
yay

**Pull request merged**